roles/contiv_network/tasks/k8s_tasks.yml

```diff
@@ -5,20 +5,30 @@
   with_items:
     - defaults
 
+- name: ensure kubernetes dirs exist
+  file:
+    path: "{{ item }}"
+    recurse: yes
+    state: directory
+  with_items:
+    - "{{ k8s_manifests_dir }}"
+    - "{{ k8s_cert_dir }}"
+    - "{{ k8s_net_plugin_config_dir }}"
+
 - name: copy contiv cni bin to k8s plugin dir
   copy:
     src: "/usr/bin/contiv/netplugin/contivk8s"
     dest: "{{ k8s_net_plugin_bin_dir }}/contivk8s"
     mode: u=rwx,g=rx,o=rx
     remote_src: yes
     force: yes
-  notify: restart kubernetes
+  #notify: restart kubernetes
```

Comment:
@rhim: We'll have to disable this for now. We won't support the installation of kubernetes with these playbooks for the time being.